### DISCUSSION TITLE
"Prior knowledge can not be converted"

### DISCUSSION THREAD

**Discussion by ninakahlo** [asreview/asreview#1808] [Q&A]
Hello everyone,

while screening articles, I accidentally marked five texts as “relevant”, but they are actually “irrelevant”. I was able to convert one of them without any problems, but not the other four. The following message appears: "Prior knowledge can not be converted".

Does anyone know why this message appears and how I can still convert the remaining four articles?

Many thanks for your help!

**Comment by jteijema** [comment]
Hi @ninakahlo !

This message should appear only when the articles were selected as prior knowledge during the setup phase of the asreview project. Can you verify this is the case? Or did you set up the project and label these documents during the regular screening?

If these records were part of the prior knowledge selected during the setup, then you will need to create a new project with the data. We want to avoid changing the prior knowledge, as this would give major issues for reproducibility, one of our main concerns :).

In either case, you could export the project data including labels. Manually relabel the 4 documents that should be changed to irrelevant, and use the dataset to start up a new project.

Jelle

**Reply by J535D165** [reply]
Good news: It will be possible to change prior knowledge in ASReview LAB 2.0, scheduled for later this year. 

For now, the solution proposed by @jteijema is the simpliest workaround.
